### PR TITLE
Add the localhost's IP address for Genymotion emulators

### PR DESCRIPTION
**Pull request by SubhrajyotiSen**
Updated the README.md for HelloEndPoints to include the localhost's IP address for Genymotion emulators in EndpointsAsyncTask to make it easier for devs.

### CODE REVIEW

**Comment by SubhrajyotiSen** [comment]
Any updates on this?

File: HelloEndpoints/README.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ class EndpointsAsyncTask extends AsyncTask<Pair<Context, String>, Void, String>
                     new AndroidJsonFactory(), null)
                 // options for running against local devappserver
                 // - 10.0.2.2 is localhost's IP address in Android emulator
+                // - 10.0.3.2 is localhost's IP address for Genymotion emulator
```
Comment (elharo) [inline HelloEndpoints/README.md]: nit: for --> in

Comment (SubhrajyotiSen) [inline HelloEndpoints/README.md]: Thanks for pointing that out. I have updated the PR

**Comment by loosebazooka** [comment]
This should be documented by the Genymotion team, not us.

**Pull request closed without merging**